deeppavlov/skills/dsl_skill/context.py

```diff
@@ -0,0 +1,35 @@
+from typing import Optional, Union
```

Comment:
It would be cool if you added your copyright and Apache 2.0 license to every `.py` file